streamer/src/nonblocking/quic.rs

```diff
@@ -692,6 +699,13 @@ async fn packet_batch_sender(
 
                 total_bytes += packet_batch[i].meta().size;
 
+                if let Some(signature) =
+                    signature_if_should_track_packet(&packet_batch[i]).unwrap_or(None)
```

Comment:
```suggestion
                    signature_if_should_track_packet(&packet_batch[i]).ok().flatten()
```

discarding the error feels bad either way we spell it tho

Reply:
done